website/docs/d/is_security_group_rule.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_is_security_group_rule"
+description: |-
+  Get information about is_security_group_rule
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm_is_security_group_rule
+
+Provides a read-only data source for is_security_group_rule. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
```

Comment:
Refer the resource of security group rule and add the reference here

Reply:
done.

Reply:
done.